dataactcore/models/jobTrackerInterface.py

```diff
@@ -275,3 +275,6 @@ def getSubmissionStatus(self,submissionId):
         if statuses["finished"] == len(jobIds)-skip_count: # need to account for the jobs that were skipped above
             return "finished"
         return "unknown"
+
+    def getSubmissionsByUserId(self, user_id):
+        return self.session.query(Submission).filter(Submission.user_id == user_id).all()
```

Comment:
This is an existing function in dataactbroker.handlers.jobHandler.  I'm fine with moving it here for use in the cgac script, but we should delete the one in jobHandler to avoid the duplication.  Since jobHandler inherits this class and the function is identical except for user_id vs userId and the missing docstring, we shouldn't need to make any other changes after deleting the jobHandler one (and if you could move the docstring over, that would be helpful).